recipes/ingeoml/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "IngeoML" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/IngeoML-{{ version }}.tar.gz
+  sha256: 38bcd14a51fa2579e8db81da5bf5bb13f3f53b69c35a59bc47a7d62a7c939c5c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
For `noarch: python`, restrict the python versions to something like `python >=3.7` both here and under `run`.